sdk/elasticsan/Azure.ResourceManager.ElasticSan/src/autorest.md

```diff
@@ -91,12 +92,12 @@ rename-mapping:
   SourceCreationData: ElasticSanVolumeDataSourceInfo
   SnapshotCreationData: SnapshotCreationInfo
   DiskSnapshotList: DiskSnapshotListContent
-  PolicyState: DeleteRetentionPolicyState
-  PreValidationResponse: PreValidationResult
-  VolumeNameList: VolumeNameListContent
-  XMsDeleteSnapshots: DeleteSnapshotsUnderVolume
-  XMsAccessSoftDeletedResources: AccessSoftDeletedVolume
-  XMsForceDelete: ForceDeleteVolume
+  PolicyState: ElasticSanDeleteRetentionPolicyState
+  PreValidationResponse: ElasticSanPreValidationResult
+  VolumeNameList: ElasticSanVolumeNameListContent
+  XMsDeleteSnapshots: ElasticSanDeleteSnapshotsUnderVolume
+  XMsAccessSoftDeletedResources: ElasticSanAccessSoftDeletedVolume
+  XMsForceDelete: ElasticSanForceDeleteVolume
```

Comment:
SAA

Reply:
`XMsForceDelete` is a bad name should be fixed